src/Tests/dotnet.Tests/CommandTests/CompleteCommandTests.cs

```diff
@@ -327,7 +327,6 @@ public void GivenDotnetToolInWithPosition()
             var expected = new[] {
                 "install",
                 "uninstall",
-                "--info"
```

Comment:
I'll run the test change by @dsplaisted since he has the most context here.

Reply:
I think this was just a bug in the old completions that was fixed in library updates. At the position the completion was requested (dotnet tool in|), where | is the cursor, `--info` isn't a valid option. `--info` is only valid on the root `dotnet` command.